Mathlib/MeasureTheory/Integral/Eval/Basic.lean

```diff
@@ -0,0 +1,149 @@
+/-
+Copyright (c) 2025 Etienne Marion. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Etienne Marion
+-/
+import Mathlib.MeasureTheory.Constructions.Pi
+import Mathlib.MeasureTheory.Integral.Bochner.ContinuousLinearMap
+
+/-!
+# Integrability in a product space
+
+We prove that `f : (i : ι) → X → E i` is in `Lᵖ` if and only if for all `i`, `f i` is in `Lᵖ`.
+We do the same for `f : X → (E × F)`.
+-/
+
+section aux
```

Comment:
Could you find a better place for the statements in that aux section? They have nothing to do with measure theory.